### DISCUSSION TITLE
Contest: lsp-mode welcome page screenshot

### DISCUSSION THREAD

**Discussion by yyoncho** [emacs-lsp/lsp-mode#3164] [Ideas]
Seems like our current welcome page screenshot is not fancy enough and we have to update it.
Rules:

1. Post screenshot/gifs of lsp-mode
2. Most upvoted in next 10 days wins and the screenshot/gif will be used for lsp-mode start page

**Comment by ericdallo** [comment]
**server**: clojure-lsp
**features**: headerline, diagnostics, signature, project-diagnostics, modeline code-actions, hover and code lenses

![image](https://user-images.githubusercontent.com/7820865/109724882-0c30da00-7b8f-11eb-8420-fffe9c7262eb.png)

**Comment by ericdallo** [comment]
@yyoncho it seems I'm the winner 🤣

**Comment by brotzeit** [comment]
My emacs doesn't look very fancy so I wasn't able to participate =)
